discord-scripts/compact-embeds.ts

```diff
@@ -0,0 +1,40 @@
+import { PartialMessage, Client, Message, EmbedBuilder } from "discord.js"
+
+async function compactGithubEmbeds(message: Message<boolean> | PartialMessage) {
+  if (message.author === null || message.author === undefined) {
+    return
+  }
+
+  if (!message.author.bot) {
```

Comment:
we can remove a level of nesting by changing this to

```typescript
if (message.author.bot) {
  return
}
```

I became convinced of the value of this after watching https://www.youtube.com/watch?v=CFRhGnuXG-4